website/docs/tutorials/sharepoint.md

```diff
@@ -46,6 +46,98 @@ Let's walk through how OpenConnector can automatically publish WOO documents fro
 
 This automated flow ensures that WOO documents are consistently and accurately published while maintaining security and compliance requirements.
 
+## How to configure SharePoint authentication
+
+### Prerequisites
+- A Microsoft SharePoint instance managed with Microsoft Entra.
+- A Nextcloud instance with OpenConnector installed.
+- A valid X.509 certificate and the private key.
+- A Linux command line (or Git Bash on Windows) for generating values later in the process.
+
+### Setting up Microsoft Entra
+
+#### Create an app
+First, we need to register an appcreate an app registration in Microsoft Entra under ‘Identity’ → ‘Applications’ → ‘App registrations’. Give your app registration an appropriate name
+![](images/Entra_AppRegistration.png)
+
+This app registration needs API permissions for Sharepoint. We recommend this to limit this to AllSites.Read and Sites.Read.All.
+
+Permissions are managed under the tab ‘API Permissions’. Click ‘Add’ and in the overlay click ‘Sharepoint’, ‘Application permissions’, and check the checkbox for ‘Sites.Read.All’, and click ‘Delegated permissions’, and check the checkbox ‘AllSites.Read’. Sites.
+
+![](images/Entra_AddPermission_1.png)
+![](images/Entra_AddPermission_2.png)
+![](images/Entra_AddPermission_3.png)
+![](images/Entra_AddPermission_4.png)
```

Comment:
![image](https://github.com/user-attachments/assets/5205df13-384e-4840-a981-fe8a54339dfc)
Could you make some of these images bigger and more readable?